Library/Homebrew/formula.rb

```diff
@@ -2418,9 +2418,7 @@ def mirror(val)
     # tell you the currently valid value.
     #
     # <pre>sha256 "2a2ba417eebaadcb4418ee7b12fe2998f26d6e6f7fda7983412ff66a741ab6f7"</pre>
-    Checksum::TYPES.each do |type|
-      define_method(type) { |val| stable.send(type, val) }
-    end
+    define_method(:sha256) { |val| stable.sha256 val }
```

Comment:
```suggestion
    delegate sha256: :stable
```

Reply:
This gives the following error:
 `Error: unknown keyword: sha256`

Reply:
@iMichka Try to add `:sha256,` to here instead:

https://github.com/Homebrew/brew/blob/a65ba2fbfedca0ace75af69666848258916e677e/Library/Homebrew/formula.rb#L341-L350

Reply:
Missing `extend Forwardable` maybe?

Reply:
It's using `delegate` elsewhere and I actually think it should be forwarded to the current spec anyway.

Reply:
This method definition is in the `class << self` block. I see that there is no other delegation there, so I guess this is not possible.

Reply:
`class << self` possibly needs `extend Forwardable`

Reply:
> It's using `delegate` elsewhere and I actually think it should be forwarded to the current spec anyway.

No, I think this is correct and should be using `stable`

Reply:
I used the same syntax as def mirror above. Works for me.
We can have a second PR to make it forwardable and use delegate if you want.